example/src/client/client.ts

```diff
@@ -0,0 +1,21 @@
+import { ErrorFormat, LogDefinition, LogLevel, YouleapClientOptions } from '../types/index';
+import { CatalogDelegate } from './bases';
+
+export class YouleapClient {
+  private errorFormat?: ErrorFormat;
+  private log?: Array<LogLevel | LogDefinition>;
+  private accessToken?: string;
+
+  constructor(args?: YouleapClientOptions) {
+    this.errorFormat = args?.errorFormat;
+    this.log = args?.log;
+    this.accessToken = args?.accessToken;
+  }
+
+  public db = {
+    catalog: new CatalogDelegate(this.accessToken),
+  };
+
+  public function = {};
+  public api = {};
```

Comment:
What is it?
Can we type it?

Reply:
It's a placeholder for feature features in youleap. for example, a way to invoke a youleap sub-function from and get its result from the SDK.

Reply:
👍